repository-data/site/src/main/resources/hcm-config/hst/configurations.yaml

```diff
@@ -115,6 +115,12 @@ definitions:
           /base-banner:
             jcr:primaryType: hst:template
             hst:renderpath: webfile:/freemarker/hee/layout/banner.ftl
+          /newslisting-main:
```

Comment:
Could we remove `newslisting-main` as we are not using it ?

Reply:
Done.